extra-network/netatalk/autobuild/build

```diff
@@ -0,0 +1,6 @@
+abinfo "Configuring..."
+./configure --with-libevent-header=/usr/include --with-libevent-lib=/usr/lib/libevent.so --with-tbb=no
```

Comment:
Two stylistic issue here, the latter is optional:

- You should invoke `configure` with an absolute path.
- If you would like, it's easier to read and manage parametres when they are broken by one per line, e.g.

```
"$SRCDIR"/configure \
    --with-libevent-header=/usr/include \
    --with-libevent-lib=/usr/lib/libevent.so \
    --with-tbb=no
```

Reply:
I should also question the reason why you have chosen to write a `build` script, when this package is in fact an Autotools source package. Use `ABTYPE=autotools`, and put the extra parametres in `AUTOTOOLS_AFTER=` instead, e.g.

```
AUTOTOOLS_AFTER="--with-libevent-header=/usr/include \
    --with-libevent-lib=/usr/lib/libevent.so \
    --with-tbb=no
```

`ABTYPE=autotools` does not have to be specified, since Autobuild3 is capable of detecting and determining the best ABTYPE for each source package.

Reply:
FYI: I copied this script from extra-network/dhcpcd/autobuild/build which wrote:

```
./configure --prefix=/usr --sysconfdir=/etc --localstatedir=/var --libexecdir=/usr/libexec --rundir=/run --dbdir=/var/lib/dhcpcd
```

Reply:
Removed `build` script and added `AUTOTOOLS_AFTER` field.